docs/articles/documentation/concepts/browsers.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: docs
+title: Browsers
+permalink: /documentation/concepts/browsers.html
+---
+# Browsers
+
+This topic lists browsers supported by TestCafe and describes how to use various browser features.
+
+* [Browser Support](#browser-support)
+* [Test in Headless Mode](#test-in-headless-mode)
+* [Use Chromium Device Emulation](#use-chromium-device-emulation)
+* [User Profiles](#user-profiles)
+
+## Browser Support
+
+### Officially Supported Browsers
+
+While TestCafe is designed to support most modern browsers, there are a limited number
+of *officially supported* browsers against which TestCafe is actively tested.
```

Comment:
Can we say:
While TestCafe is designed to support most modern browsers, there are a limited number
of *officially supported* browsers that we have actively tested.

Reply:
I don't think we can say that we 'have tested browsers'. We tested TestCafe in conjunction with these browsers, not them.

Reply:
I'm not quite sure what you mean. Can you propose a rewrite?

Reply:
I'm not sure I can propose a rewrite, because we struggled to come up with even this phrasing. What we wanted to say is:

TestCafe runs tests in the browser. Due to its architecture, TestCafe can theoretically work in any modern browser that complies with recent standards. But, we cannot say for sure because we are not able to check how TestCafe works with just any browser out there. So we ensure that TestCafe works with the most popular browsers. These browsers are *officially supported*, and we say that *TestCafe is actively tested against these browsers*. I'm not sure but I recall we consulted with Julian regarding this phrase.

Reply:
I see. How about:
While TestCafe is designed to support most modern browsers, there are a limited number
of *officially supported* browsers. TestCafe is actively tested with these browsers.